akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -923,7 +923,7 @@ private[remote] object ArteryTransport {
 
   val ProtocolName = "artery"
 
-  val Version = 0
+  val Version: Byte = 0
```

Comment:
it probably doesn't matter much, but I guess we will use this to compare if the received envelope is of a specific version, then we can perhaps keep this as `Int`. `Int` and `Byte`.
```scala
scala> val b: Byte = 12
b: Byte = 12

scala> val i = 12
i: Int = 12

scala> b == 12
res1: Boolean = true
```